src/features/shared-link-settings-modal/AllowDownloadSection.js

```diff
@@ -35,23 +38,38 @@ const AllowDownloadSection = ({
         </div>
     );
 
+    const label = classification ? `"${classification}"` : null;
+    const tooltipMessage = { ...messages.directDownloadBlockedByAccessPolicy, values: { label } };
+    const downloadWrapperClass = isDirectLinkUnavailableDueToAccessPolicy
+        ? 'bdl-AllowDownloadSection-wrapper--disabled'
+        : null;
+
     if (isDownloadAvailable) {
         return (
-            <div>
+            <div className={downloadWrapperClass}>
                 <hr />
-                <Fieldset title={<FormattedMessage {...messages.allowDownloadTitle} />}>
-                    <Checkbox
-                        isChecked={isDownloadEnabled}
-                        isDisabled={!canChangeDownload}
-                        label={<FormattedMessage {...messages.allowDownloadLabel} />}
-                        name="isDownloadEnabled"
-                        onChange={onChange}
-                        {...downloadCheckboxProps}
-                    />
-                    {(isDirectLinkAvailable || isDirectLinkUnavailableDueToDownloadSettings) && isDownloadEnabled
-                        ? directLinkSection
-                        : null}
-                </Fieldset>
+                <Tooltip
+                    isDisabled={!isDirectLinkUnavailableDueToAccessPolicy}
+                    text={<FormattedMessage {...tooltipMessage} />}
+                    position="middle-left"
+                >
+                    <Fieldset
+                        disabled={isDirectLinkUnavailableDueToAccessPolicy}
+                        title={<FormattedMessage {...messages.allowDownloadTitle} />}
+                    >
+                        <Checkbox
+                            isChecked={isDownloadEnabled}
+                            isDisabled={!canChangeDownload || isDirectLinkUnavailableDueToAccessPolicy}
+                            label={<FormattedMessage {...messages.allowDownloadLabel} />}
+                            name="isDownloadEnabled"
+                            onChange={onChange}
+                            {...downloadCheckboxProps}
+                        />
+                        {(isDirectLinkAvailable || isDirectLinkUnavailableDueToDownloadSettings) && isDownloadEnabled
```

Comment:
can you set this as a constant? getting long here

Reply:
Also once made into a const, you can remove the ?: